### ISSUE TITLE
Cat vs Dog

### ISSUE THREAD

**Issue by Miladiouss** [bamos/densenet.pytorch#17]
I have slightly modified your algorithm and have adapted it for two classes (k = 12, reduction = 0.5, bottleneck = True). When I train it on Cat and Dog images from CIFAR-10, I only go as high as 82% validation accuracy. Is that what you get as well? Or you get something closer to accuracy for all 10 classes, i.e. > %95?
